pkg/morph/event/rolemanagement/designate.go

```diff
@@ -40,6 +40,6 @@ func ParseDesignate(e *state.ContainedNotificationEvent) (event.Event, error) {
 
 	return Designate{
 		Role:   noderoles.Role(bi.Int64()),
-		TxHash: e.Container,
+		TxHash: util.Uint256{},
```

Comment:
I'm not excited about this one because I'd expect this "real->fake" translation at some other layer dealing with the event. But it should work.